pkg/cloudproviders/gcp/cred_manager_impl.go

```diff
@@ -0,0 +1,76 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/pkg/secretinformer"
+	"github.com/stackrox/rox/pkg/sync"
+	"golang.org/x/oauth2/google"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	cloudCredentialsKey = "credentials"
```

Comment:
I thought we were going to choose something slightly more specific for the hierarchy, such as an account ID or something like that?

Reply:
I thought about that again and I don't think it's necessary to require the key to carry any specific meaning. If we want to extend the implementation to multiple keys in the future, we can let the customer choose any identifiers they like. These are then used in the UI to select the correct configuration for each integration.